pkg/apis/config/features.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	cm "knative.dev/pkg/configmap"
+)
+
+type Flag string
+
+const (
+	// FeaturesConfigName is the name of the ConfigMap for the features.
+	FeaturesConfigName = "config-features"
+
+	Enabled  Flag = "Enabled"
+	Allowed  Flag = "Allowed"
+	Disabled Flag = "Disabled"
```

Comment:
If only those 3 values are allowed, maybe we could use a [`iota`](https://github.com/uber-go/zap/blob/master/zapcore/level.go#L31-L78) here? Feel free to ignore if you think it's too messy.

Reply:
No strong opinion here. But since those flags are used to parse strings from a ConfigMap, defining their string representation here allow us to not have to deal with the `string` -> `Flag` separately.

Reply:
OK, sounds reasonable